R/get_coverage_wcs.R

```diff
@@ -20,7 +20,7 @@
 #'   - `"omw"`: orthophotomosaic winter images Flanders
 #'   - `"dtm"`: digital terrain model Flanders
 #'   - `"dsm"`: digital surface model Flanders
-#' See https://metadata.vlaanderen.be/metadatacenter/srv/dut/catalog.search#/search?keyword=OGC:WCS for more information # nolint
+#' See [metadata Vlaanderen](https://metadata.vlaanderen.be/metadatacenter/srv/dut/catalog.search#/search?keyword=OGC:WCS) for more information # nolint
```

Comment:
You can specify the linter to ignore. This has the benefit that you keep checking the other linters.
```suggestion
#' See [metadata Vlaanderen](https://metadata.vlaanderen.be/metadatacenter/srv/dut/catalog.search#/search?keyword=OGC:WCS) for more information # nolint line_length_liner;
```

Reply:
You must add a colon right after `nolint`, otherwise all linters will still be excluded. Also the linters must be comma-separated and at the end there must be a point AFAIK.

Reply:
`# nolint: line_length_liner, xxxx.`

Reply:
I've updated my suggestion.

Reply:
see [efa54d6](https://github.com/inbo/inbospatial/pull/1/commits/efa54d606d348cbb4514f97dbd014021634ce692)